draft-ietf-mls-protocol.md

```diff
@@ -2230,65 +2230,49 @@ that the path from P to the root may contain some blank nodes that are not
 fixed by P's Parent Hash. However, for each node that has an HPKE key, this key
 is fixed by P's Parent Hash.
 
-Finally, `original_sibling_tree_hash` is the original tree hash of S. The
-original tree hash corresponds to the tree hash of S the last time P was
-updated. It can be computed as the tree hash of S in the ratchet tree modified
-the following way:
+Finally, `original_sibling_tree_hash` is the tree hash of S in the ratchet tree
+modified as follows:
 
-* reset the leaves in P.unmerged_leaves to blanks
-* remove P.unmerged_leaves from all unmerged_leaves lists
-* truncate the ratchet tree as described in {{remove}}
+* Extend the subtree of S until it is full (see {{adding-and-removing-leaves}}).
```

Comment:
Suggest "Extend the subtree of S by adding blank leaves".  Also, since we don't talk much about "full", a reminder might be helpful here, say "(i.e., until its size is a power of 2)".

Reply:
Done. Hope it's better now.